### ISSUE TITLE
import failure when importing `AdcircRun` from `adcircpy.driver.driver`

### ISSUE THREAD

**Issue by ghost** [oceanmodeling/adcircpy#73]
The following error happens only in the Github Action (~~I can't reproduce it on my local Windows~~)

https://github.com/noaa-ocs-modeling/CoupledModelDriver/runs/2272056215?check_suite_focus=true

```
________________ ERROR collecting tests/test_configurations.py _________________
ImportError while importing test module '/home/runner/work/CoupledModelDriver/CoupledModelDriver/tests/test_configurations.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/hostedtoolcache/Python/3.9.2/x64/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_configurations.py:7: in <module>
    from coupledmodeldriver.configuration import (
coupledmodeldriver/configuration.py:9: in <module>
    from adcircpy.driver.driver import AdcircRun
/opt/hostedtoolcache/Python/3.9.2/x64/lib/python3.9/site-packages/adcircpy/__init__.py:6: in <module>
    from adcircpy.driver.driver import AdcircRun
E   ModuleNotFoundError: No module named 'adcircpy.driver'
```

**Comment by ghost** [comment]
I was able to reproduce this issue by installing `adcircpy` `v1.0.26` fresh, and it does not have a `driver` module

**Comment by ghost** [comment]
fixed with e8c74a21701d89f163241a7f4ac9ea291a5088a9, v1.0.26
